WalletWasabi/Wallets/CpfpInfoProvider.cs

```diff
@@ -198,11 +194,8 @@ private async Task<CpfpInfo> GetCpfpInfoAsync(uint256 txid, CancellationToken ca
 		using CancellationTokenSource timeoutCts = new(TimeSpan.FromSeconds(20));
 		using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, timeoutCts.Token);
 
-		var response = await _httpClient.SendAsync(
-			HttpMethod.Get,
-			$"v1/cpfp/{txid}",
-			null,
-			linkedCts.Token).ConfigureAwait(false);
+		var httpClient = _httpClientFactory.CreateClient($"mempool.space-{txid}");
+		var response = await httpClient.GetAsync( $"v1/cpfp/{txid}", linkedCts.Token).ConfigureAwait(false);
```

Comment:
This is broken (found by @Kruwed )
The `HttpClient` should have as `BaseUri` `_uri`, or the absolute URI should be passed here

Reply:
Solved